packs/mmonit/actions/get_uptime_host.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: get_uptime_hosts
+runner_type: run-python
+description: Returns services uptime for a particular host.
+enabled: true
+entry_point: get_uptime_hosts.py
+parameters:
+  host_id:
+    type: integer
+    description: 'ID of the host'
+    required: true
+  uptime_range:
+    type: integer
+    description: 'The range id. Possible values: 0=today, 5=week, 6=month, 7=three months, 8=six months, 9=year and 12=custom range. Default is 0=today'
+    required: false
+    enum:
```

Comment:
It might be a bit more user-friendly to use actual string values here and in other places where enums are used (today, week, month, etc.) instead of the numeric values. Then we could map those string values to numeric values in the code.

Anyway, that's not a blocker or anything, just an optimization we can do in the future :)